src/trackers/COMMON.py

```
@@ -46,7 +47,7 @@ async def async_input(self, prompt=False):
         user_input = await asyncio.to_thread(input)
         return user_input.strip()
 
-    async def create_torrent_for_upload(self, meta, tracker, source_flag, torrent_filename="BASE", announce_url=None):
+    async def create_torrent_for_upload(self, meta: dict[str, Any], tracker: str, source_flag: str, torrent_filename: str = "BASE", announce_url: str = None):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)